projects/components/src/confirmation/confirmation-modal.component.ts

```diff
@@ -0,0 +1,55 @@
+import { ChangeDetectionStrategy, Component, Inject } from '@angular/core';
+import { ButtonRole } from '../button/button';
+import { ModalRef, MODAL_DATA } from '../modal/modal';
+
+@Component({
+  selector: 'ht-confirmation-modal',
+  styleUrls: ['./confirmation-modal.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  template: `
+    <div class="confirmation-modal">
+      <div class="description">
+        {{ this.descriptionText }}
+      </div>
+      <div class="controls">
+        <ht-button [label]="this.cancelButtonLabel" role="${ButtonRole.Tertiary}" (click)="this.onCancel()"></ht-button>
+        <ht-button
+          [label]="this.confirmButtonLabel"
+          [role]="this.confirmButtonRole"
+          (click)="this.onConfirmation()"
+        ></ht-button>
+      </div>
+    </div>
+  `
+})
+export class ConfirmationModalComponent {
+  private static readonly DEFAULT_CONFIRM_LABEL: string = 'OK';
```

Comment:
Shouldn't default be 'Confirm'?

Reply:
Probably - that feels so formal though! We can change this once we start actually using it and get a sense of which is a better fit.

Reply:
All our existing usecases are for 'Confirm'. Hence the suggestion.

Reply:
Fair enough